rocketpy/Rocket.py

```diff
@@ -420,9 +423,10 @@ def addTail(self, topRadius, bottomRadius, length, distanceToCM):
 
         # Calculate clalpha
         clalpha = -2 * (1 - r ** (-2)) * (topRadius / rref) ** 2
+        self.cldata = Function(lambda x: clalpha*x, 'Alpha (rad)', 'Cl', interpolation='linear')
```

Comment:
I am not sure this works as you intended. Using `self.cldata` sets the Rocket's attribute `cldata`, not `cldata` for the tail only.

Reply:
You are right, that explains why the corrections is so strong. I'll fix it.